testscript/testscript.go

```diff
@@ -200,13 +200,21 @@ func (t tshim) Verbose() bool {
 // RunT is like Run but uses an interface type instead of the concrete *testing.T
 // type to make it possible to use testscript functionality outside of go test.
 func RunT(t T, p Params) {
-	glob := filepath.Join(p.Dir, "*.txt")
-	files, err := filepath.Glob(glob)
-	if err != nil {
+	entries, err := os.ReadDir(p.Dir)
+	if os.IsNotExist(err) {
+	} else if err != nil {
```

Comment:
```suggestion
	if err != nil && !os.IsNotExist(err) {
```
?

Reply:
The old globbing code would return zero matches due to a missing directory as `nil, nil`, but `os.ReadDir` errors - so to match the old behavior, I ignore "does not exist" errors for the directory. This makes one of the tests happy, and IMO it's nicer to say "no txtar files found in directory" rather than a platform specific "open directory: not found".

Reply:
Added the comment. I kept the logic as it is now, because I can more clearly add the comment about IsNotExist.